core/src/main/java/hudson/Proc.java

```diff
@@ -64,6 +64,11 @@
 public abstract class Proc {
     protected Proc() {}
 
+    /**
+     * Indicates that the process should not be killed on interruption.
+     */
```

Comment:
`@since TODO`. Protected fields are a part of API

Reply:
Done